hastory-test-utils/test/HastorySpec.hs

```diff
@@ -6,13 +6,32 @@ module HastorySpec
 
 import TestImport
 
+import qualified Data.Text as T
+import Safe (tailSafe)
+
 import Hastory.Gen ()
+import Hastory.SuggestAlias
 import Hastory.Types
 
 spec :: Spec
-spec =
+spec = do
     describe "Entry" $ do
         eqSpec @Entry
         jsonSpecOnValid @Entry
         arbitrarySpec @Entry
         genValiditySpec @Entry
+    describe "commandPrefixes" $ do
+        it "Lists only prefixes of the command, ignoring whitespace" $
+            forAll genValid $ \e ->
```

Comment:
`forAll genValid` should usually be replaced with `forAllValid`, so you get the shrinking as well.